config/projects/chef.rb

```diff
@@ -40,20 +40,14 @@
 
 override :bundler,        version: "1.7.2"
 override :ruby,           version: "2.1.6"
-######
-# Ruby 2.1.3 is currently not working on windows due to:
-# https://github.com/ffi/ffi/issues/375
-# Enable below once above issue is fixed.
-# override :'ruby-windows', version: "2.1.3"
-# override :'ruby-windows-devkit', version: "4.7.2-20130224-1151"
-override :'ruby-windows', version: "2.0.0-p451"
-######
+
+override :'ruby-windows', version: "2.0.0-p645"
 override :rubygems,       version: "2.4.4"
 
 # Chef Release version pinning
 #override :chef, version: "12.4.1"
 #override :ohai, version: "8.5.0"
-override :'openssl-windows', version: '1.0.0r'
+
```

Comment:
Because that's the default openssl-windows version now.  I can leave it as that explicitly if you'd like.  Both chef and chef-dk should be using that version.